requests-for-comments/commnunity-2024.md

```diff
@@ -0,0 +1,47 @@
+Up to now, much of our thinking for the community section of the website has centered on creating a Hacker News-like experience where content drives the community. I think this is the wrong direction. This issue outlines why and an alternative vision of the community.
+
+## Why not the current direction?
+
+Content, both our own and good content curated _by us_ can still form an important part of our community, but:
+
+1. There are already numerous places to share content about building startups / engineering topics generally. Why should people post it to ours and not another, more established channel?  
+
+2. The attraction of our content comes from our strong POV and curation. Outsourcing curation to the community undermines a strength we already have.
+
+3. If it were successful, it would create a significant moderation challenge that would suck up a bunch of time. Abuse and self-promotion will happen and we'll have to manage that.
+
+4. There's a significant risk our own content would be drowned out by community contributions and submissions to the point.
```

Comment:
Again, solvable problem. We could pin ours to the top etc.

Reply:
Solvable with weighting